bundles/org.eclipse.vorto.editor.datatype/src/org/eclipse/vorto/editor/datatype/Datatype.xtext

```diff
@@ -111,6 +111,14 @@ ObjectPropertyType:
 	type = [Type|QualifiedName]
 ;
 
+ComplexPrimitivePropertyType:
+	DictionaryPropertyType
+;
+
+DictionaryPropertyType:
+	{DictionaryPropertyType} 'Dictionary' ('[' keyType = PrimitiveType ',' valueType = PrimitiveType ']')?
```

Comment:
Why is the keytype and valuetype only restricted to PrimitiveTypes? 
I can imagine that a user would also like to define a constraint for Object Types , like in a ordinary map.What do you think?